monasca-log-transformer/Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM logstash:2-alpine
+
+# To force a rebuild, pass --build-arg REBUILD="$(DATE)" when running
+# `docker build`
+ARG REBUILD=1
+
+ENV CONFIG_TEMPLATE=true \
+  KAFKA_URI=kafka:9092 \
+  ZOOKEEPER_URI=zookeeper:2181 \
+  KAFKA_WAIT_FOR_TOPICS=log-transformed,log
+
+ARG REBUILD_DEPENDENCIES=1
+RUN apk add --no-cache python py2-pip py2-jinja2 && \
+  apk add --no-cache --virtual build-dep \
+  python-dev make g++ linux-headers && \
+  pip install pykafka && \
+  apk del build-dep
+
+ARG REBUILD_CONFIG=1
+COPY log-transformer* /etc/monasca/
+COPY template.py start.sh kafka_wait_for_topics.py /
+
+CMD ["/start.sh"]
```

Comment:
Personally I tend to like CMD since you can more easily override with e.g. `sh` if needed for test/debugging. Plus eventually I'd like to see our containers all use [tini](https://github.com/krallin/tini) which recommends setting the entrypoint to a different executable.